latch/auth/pkce.py

```diff
@@ -87,7 +88,7 @@ def __enter__(self, *args):
     def __exit__(self, *args):
         ...
 
-    def construct_challenge(self) -> (str, str):
+    def construct_challenge(self) -> Tuple[str, str]:
```

Comment:
any reason for this

Reply:
not in particular - the linter that im using enforces super explicit type annotations so i changed it. i dont really have any strong opinions on this so i dont mind changing it back